pavelib/quality.py

```diff
@@ -758,11 +758,15 @@ def _get_xsscommitlint_count(filename):
 
 @task
 @needs('pavelib.prereqs.install_python_prereqs')
+@cmdopts([
+    ("report-dir=", "r", "Directory in which to put PII reports, defaults to reports/pii_report/"),
+])
 @timed
 def run_pii_check(options):  # pylint: disable=unused-argument
     """
     Guarantee that all Django models are PII-annotated.
     """
+    report_dir = getattr(options, 'report_dir', u'reports/pii_report/')
```

Comment:
nit: for consistency's sake, could you set the report directory similar to the other quality checks, i.e. https://github.com/edx/edx-platform/blob/2a73593f79548723de59537708113acc82e13588/pavelib/quality.py#L367 ? It will just make future debugging/refactors nicer.